cmd/runj/create.go

```diff
@@ -71,16 +80,27 @@ command(s) that get executed on start, edit the args parameter of the spec.`,
 			return nil
 		},
 	}
-	consoleSocket := create.Flags().String(
+	flags := create.Flags()
+	flags.StringVar(
+		&bundle,
+		"bundle",
+		"",
+		`path to the root of the bundle directory, defaults to the current directory`)
```

Comment:
Do you have a specific need for runj to support CWD here as the default or are you more interested in just adding support for `--bundle`?  CWD does not currently work with your PR (the `jail` utility requires an absolute pathname and there's no current code to compute that).

I'm going to remove support for CWD for now while I'm working on this code and we can add it back in a separate PR if you need it.